### ISSUE TITLE
Is AdGuardExtra different from AdGuard "Advanced Protection" in Safari ?

### ISSUE THREAD

**Issue by aayushbhatia06** [AdguardTeam/AdGuardExtra#548]
### Please answer the following questions for yourself before submitting a question.

- [X] I am running the latest version
- [X] I checked to make sure that this issue has not already been filed

### Ask a question

Hi there!

I have a seemingly simple question but with no clear answer anywhere that I could see. I want to know if AdGuardExtra (in UserScript form, as that's the only option for Safari) is different from the "Advanced Protection" setting in Adguard iOS or the AdGuard for Safari extension (the one that reads - "This extension performs two important functions: it adds an icon to Safari toolbar which can be used to manage AdGuard settings, and it also applies advanced blocking rules which are not supported by Safari Content Blocking API.") 

In other words, do we need both or either? Thanks!

**Comment by jellizaveta** [comment]
AdGuard Extra works on a limited number of sites (only where blocker rules cannot handle ads). AdGuard iOS with Advanced Protection works on all sites. You can find out a little more about this [here](https://github.com/AdguardTeam/AdGuardExtra?tab=readme-ov-file#important).
